Cargo.toml

```diff
@@ -21,6 +21,9 @@ travis-ci = { repository = "capstone-rust/capstone-sys" }
 bindgen = { optional = true, version = "0.30.0" }
 pkg-config = { optional = true, version = "0.3" }
 cmake = {optional = true, version = "0.1.2"}
+cc = {optional = true, version = "1.0"}
+[target.'cfg(windows)'.build-dependencies]
```

Comment:
Could you put a newline here to separate the `[target.'cfg(windows)'.build-dependencies]` line from the previous line?

Reply:
No worries, I'll make the minor change myself.